be/src/util/timezone_utils.cpp

```diff
@@ -21,13 +21,43 @@
 #include <cctz/time_zone.h>
 #include <re2/stringpiece.h>
 
+#include <boost/algorithm/string.hpp>
+#include <filesystem>
+#include <string>
+
 namespace doris {
 
 RE2 TimezoneUtils::time_zone_offset_format_reg("^[+-]{1}\\d{2}\\:\\d{2}$");
+DorisCallOnce<Status> TimezoneUtils::load_timezone_once_;
+std::unordered_map<std::string, std::string> TimezoneUtils::timezone_names_map_;
 
 const std::string TimezoneUtils::default_time_zone = "+08:00";
 
+void TimezoneUtils::_load_timezone_names() {
+    std::string path;
+    const char* tzdir = "/usr/share/zoneinfo";
```

Comment:
maybe it is not right. For exmple, in macos, the dir is not right. Or in some other linux distributions.
Could we add a static map in doris's code?